### ISSUE TITLE
Snapshot does not return ID after creation

### ISSUE THREAD

**Issue by dmitryshagin** [StackPointCloud/fog-packet#21]
When creating snapshots by `connection.snapshots.create(storage_id: "some_storage_id")` it returns without any useful information.
```
<Fog::Compute::Packet::Snapshot
    id=nil,
    status=nil,
    timestamp=nil,
    created_at=nil,
    volume=nil,
    storage_id="some_storage_id"
```
It it s bug or feature? At least it's inconsisten witn device creating - here we have server's ID just after request.

**Comment by jasmingacic** [comment]
@dmitryshagin thanks for reporting the issue. It seems that `storage_id` shouldn't be mapped at all. Also to retrieve a snapshot you need the `storage_id` so you already have that information.

**Comment by edevenport** [comment]
@dmitryshagin When a snapshot is created through the API, there is no body returned in the response. It seems the `create` function is returning the snapshot object, but with no body, there are no values to populate. Usually the `model.reload` would populate those values, but there is no API request to get a single snapshot (only list all snapshots).

@jasmingacic If the API is not returning the snapshot details on POST, I don't think the library `create` function should return the object. Perhaps we return `true` if the request is successful?

**Comment by edevenport** [comment]
So it looks like the `create` function should already be returning `true`:

```
        def save
          requires :storage_id
          response = service.create_snapshot(storage_id)
          true if response.status == 202
        end
```

That's the behavior I would expect based on the API response.

**Comment by edevenport** [comment]
I agree there is some room for improvement here. I have submitted an inquiry on whether the snapshot API endpoint can be expanded with the following:

* Return snapshot properties (including ID) on POST.
* Allow GET for a single snapshot by ID.

We can decide how to proceed once I hear back.

**Comment by vielmetti** [comment]
This has been looked at by the Packet engineering team and has been planned for work but not scheduled yet.
